insights/terraform/projects/bfd/api-requests/glue.tf

```diff
@@ -25,25 +21,1306 @@ module "glue-table-api-requests" {
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_dme_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_dme_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_dme_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hha_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hha_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hha_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hospice_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hospice_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_hospice_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_inpatient_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_inpatient_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_inpatient_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_outpatient_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_outpatient_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_outpatient_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_pde_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_pde_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_pde_record_count",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_snf_duration_milliseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_snf_duration_nanoseconds",
+      "type"    = "string",
+      "comment" = ""
+    },
+    {
+      "name"    = "mdc_jpa_query_eobs_by_bene_id_snf_record_countstring",
```

Comment:
typo here that I noticed using the data just now. I don't want to change it unless we have a 100% foolproof way to ensure that the data will be preserved. It is not a big enough deal. But we should at least write a comment here about it if we don't have a way to correct it in-place.

Reply:
thinking about this more, I believe it can just be changed in glue in the schema editor and then in here, right?

Reply:
If the source of the typo was this terraform then likely the old data wouldn't have been captured and the new data would not be captured either. If the source was the Athena, then the old data was probably captured but new data would not be.

Either way, I'll figure out the source and see if there are others. Great catch!